package.json

```diff
@@ -11,10 +11,16 @@
     "kind": "plugin"
   },
   "scripts": {
-    "eslint": "eslint --max-warnings=0 './**/*.{js,jsx}'",
```

Comment:
I'm OK with removing the `--max-warnings=0` for now, but I would suggest turning it back on before the next NPM release.

I know it's a tough measure, but this is an open-source repo and if we allow eslint warnings then people without proper linting config in their IDE could contribute linting warnings without us knowing about it. I really think it's an important one.

Reply:
I definitely agree that it should be turned back on!
Why should it be before the next npm release?

Reply:
Just to time-box it, so we won't forget.

Reply:
We could have a separate workflow that could test only the changed files. I found the following action https://github.com/marketplace/actions/get-all-changed-files that we could use for this purpose.

What do you think of that? :)

Reply:
I lean towards just running eslint for all the files.
Let's just keep the `--max-warnings=0` off for now and we'll see what happends :)

Reply:
Awesome! if you are okay with the changes, could you approve them? :)

Reply:
Yeah, if you fix the build errors I'll approve :)